base/Base_compiler.jl

```diff
@@ -185,6 +185,11 @@ function Core.kwcall(kwargs::NamedTuple, ::typeof(invoke), f, T, args...)
     T = rewrap_unionall(Tuple{Core.Typeof(kwargs), Core.Typeof(f), (unwrap_unionall(T)::DataType).parameters...}, T)
     return invoke(Core.kwcall, T, kwargs, f, args...)
 end
+# for m::Method however, assume the user already looked it up correctly for kwargs but just needs the syntax rewrite to swap the positions of the invoke, kwcall, and other arguments
+function Core.kwcall(kwargs::NamedTuple, ::typeof(invoke), f, m::Method, args...)
```

Comment:
I don't know that I like this. If you already have to do the Method lookup yourself using the transformed args, it seems fair to require you to transform the args themselves also.

Reply:
That is a good point. I think this can be a separate PR if needed, since it isn't part of the core functionality of the PR. I guess it mostly brings up the question: why isn't there an API for looking up the Method without doing the argument transform yourself (which requires knowing very specific and unstable details about the lookup transform--refs `Base.bodyfunction`?), and should there be one?